amoro-format-mixed/amoro-mixed-spark/v3.2/amoro-mixed-spark-3.2/pom.xml

```diff
@@ -96,6 +96,12 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.iceberg</groupId>
+            <artifactId>iceberg-arrow</artifactId>
```

Comment:
It seems that the Iceberg spark connector dependent on iceberg-arrow in runtime scope.

Maybe we should add this dependency in amoro-format-mixed-spark-3-common in compile scope, HDYT?
Besides, we may add it into mixed format spark runtime shade rules too.